src/components/DataTable/Table.js

```diff
@@ -126,7 +127,7 @@ const Headers = ({ headers, advmode, sublists }) => (
 			{headers.map((_, idx) =>
 				<col
 					className={`data-table-col${idx}`}
-					style={{ width: (idx === headers.length - 1) ? "auto" : "1px" }}
+					style={{ width: (idx === headers.length - 1) ? "auto" : "5em", ..._.customHeaderStyle }}
```

Comment:
@petrKavulok

1] shouldnt be `..._.customHeaderStyle` `...customHeaderStyle` ?

2] Do we want to preserve the condition on `width: "auto"` ? What is the benefit of it, if default should be `5em` as you've described in the docu?

Reply:
a summary of what was discussed on stand up
1.) we're maping headers on ln127. -> the underscore `_` represents each element (not sure why it is like that, I didn't change it since i thought it was some sort of legacy code :D )

2.) `width: auto` is assigned to the last column of the table to always fill the rest of the table